### PR TITLE
php53: update bottle

### PR DESCRIPTION
**Pull request by javian**

### CODE REVIEW

File: Formula/php53.rb [hunk 1]
```diff
@@ -50,8 +50,7 @@ def php_version_path
 
   # Previous Bison and 10.9+ patches, and multi-SAPIs patch (https://pecl.php.net/~jani/patches/multi-sapi.patch) applied
   patch do
-    url "https://gist.githubusercontent.com/javian/bfcbd5bc5874ee9c539fb3d642cdce3e/raw/bf079cc68ec76290f02f57981ae85b20a06dd428/multi-sapi-5.3.29-homebrew.patch"
-    sha256 "3c3157bc5c4346108a398798b84dbbaa13409c43d3996bea2ddacb3277e0cee2"
+    url "https://gist.githubusercontent.com/javian/bfcbd5bc5874ee9c539fb3d642cdce3e/raw/48aedade9ed331ef8d68386d9b5917d6a3ef617f/multi-sapi-5.3.29-homebrew.patch"
```
Comment (yvh) [inline Formula/php53.rb]: Use this code 
`url "https://gist.githubusercontent.com/yvh/08b306fc722fbc31c446f794567dfa66/raw/ca7bad287e3af72b83fbb5f7d03186b5da2e65ae/multi-sapi-5.3.29-homebrew.patch"`

File: Formula/php53.rb [hunk 2]
```diff
@@ -50,8 +50,7 @@ def php_version_path
...
+    sha256 "91698d27ea5fe5d2a8a71a8e673f9394b7f5b400c2cc90967c8df4c38bfbbcae"
```
Comment (yvh) [inline Formula/php53.rb]: And this code 
`    sha256 "bdde5c0c1f9d2450d8d9d0a32882e1ee0008d8339eacf919e273e6f7937e7461"
`

Comment (javian) [inline Formula/php53.rb]: Thanks - I cloned it from you to keep it under my account.

**Comment by yvh** [comment]
@javian How can we resolve these errors with CI ? Let me know ;-)

**Comment by javian** [comment]
@yvh as far as I know we can't. Some of the problem comes from enabling different types of modules at the same time. The conflicts are not handled in the Formulas so when they get installed on top of eachother then it causes problems.

**Comment by javian** [comment]
thanks for the submission ! @yvh

**Comment by javian** [comment]
Not sure what happened here but now it is commited to the master branch but the PR seems flaky.

**Pull request closed without merging**